### DISCUSSION TITLE
[30 Giugno 2022] - Pipeline

### DISCUSSION THREAD

**Discussion by notedo** [CS-Swap/Architettura-degli-Elaboratori#17] [Pipeline]
### Traccia Esercizio

![esame](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/b90e61cc-8293-4524-8666-c144a7918306)
![esame1](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/5d0f05f8-1deb-4052-abfc-230e3d3f96be)
![esame3](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/50fde048-a263-49cb-9f82-2f09d77fb99a)

**Comment by thegeek-sys** [comment]
# Senza forwarding
<img width="1093" alt="Screenshot 2024-06-04 alle 09 30 27" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/55345633/6727b979-6029-409c-800e-bafad8866af3">

# Con forwarding
<img width="1093" alt="Screenshot 2024-06-04 alle 09 30 44" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/55345633/74f28d4d-ad40-4d0f-a274-d64d5a2d3c2d">

# Data hazard e control hazard
<img width="290" alt="Screenshot 2024-06-04 alle 09 31 00" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/55345633/90eecf7c-a9fe-41ab-bde4-537f5052d515">

# Istruzioni 14° ciclo di clock
<img width="242" alt="Screenshot 2024-06-04 alle 09 31 06" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/55345633/1004ffd7-0364-4560-a4c3-fb235b881d68">

**Reply by alem1105** [reply]
Vengono diverse perche' i codici hanno ordini diversi nelle foto :(

![image](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/48867964/ca57e8af-4c95-4a21-a34d-8085e9cbff96)
